packages/ontario-frontend-cli/create/skeleton/src/_includes/vendor/ontario-ca/atoms/_header_menu_open_button.njk

```diff
@@ -3,7 +3,7 @@
   aria-label="{{ 'header.labels.openMenu' | localeString }}">
   <svg class="ontario-icon" focusable="false" sol:category="primary" viewBox="0 0 24 24"
     preserveAspectRatio="xMidYMid meet">
-    <use xlink:href="#ontario-icon-menu"></use>
+    <use xlink:href="#ontario-icon-menu" id="ontario-header-menu-icon"></use>
```

Comment:
Commenting this just as an extra reminder that this fixed the menus after our migration